docs/source/using/profiles.adoc

```diff
@@ -12,7 +12,26 @@ toc::[]
 == Overview
 
 {project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+Each {project} instance can be created with different configurations e.g memory, CPU, disk-size , add-ons etc and users can switch between different instances.
+Profile feature would help users to test different configurations without deleting an existing {project} instance.
+
+Each profile is completely independent of other profiles and things like cache, add-ons and persistent config are specific to only one profile.
```

Comment:
Ok, here we talk about being independent. I think this needs to come earlier. Also I suggest using the "share nothing" term as well.